apps/test-bundles/webpackUtils.js

```diff
@@ -70,69 +70,114 @@ function createWebpackConfig(entries) {
   });
 }
 
+/**
+ * Webpack will remove any unused import as a dead code (tree shaking).
+ * Thus we are creating temporary JS files with top-level component imports
+ * and console logging them. This will ensure that the code is active
+ * and that webpack bundles it correctly.
+ */
+function createFluentNorthstarFixtures() {
+  const packageName = '@fluentui/react-northstar';
+  const distPath = path.dirname(require.resolve(packageName).replace('commonjs', 'es'));
+  const packagePath = path.resolve(distPath, 'components');
+  fs.readdirSync(packagePath).forEach(itemName => {
+    const isFolder = fs.statSync(path.join(packagePath, itemName)).isDirectory();
+
+    if (isFolder && itemName) {
+      const importStatement = `import { ${itemName} } from '${packageName}'; console.log(${itemName})`;
+      try {
+        const folderName = getFolderName(packageName);
+        const entryPath = path.join('temp/fixtures/', folderName, `${itemName}.js`);
+        fs.outputFileSync(entryPath, importStatement, 'utf-8');
+      } catch (err) {
+        console.log(err);
+      }
+    }
+  });
+}
+
 // Files which should not be considered top-level entries.
 const TopLevelEntryFileExclusions = ['index.js', 'version.js', 'index.bundle.js'];
 
+function createFluentReactFixtures() {
+  const packageName = '@fluentui/react';
+  const distPath = path.dirname(require.resolve(packageName).replace('lib-commonjs', 'lib'));
+  const packagePath = path.resolve(distPath);
+  fs.readdirSync(packagePath).forEach(itemName => {
+    const isFolder = fs.statSync(path.join(packagePath, itemName)).isDirectory();
+    const isAllowedFile = itemName && itemName.match(/.js$/) && !TopLevelEntryFileExclusions.includes(itemName);
+
+    if (isAllowedFile && !isFolder) {
+      const item = isFolder ? itemName : itemName.replace(/.js$/, '');
+      const importStatement = `import * as p from '${distPath}\${item}'; console.log(p)`;
```

Comment:
Why does this path use a backslash? Typically JS import paths use forward slashes.

Reply:
Also I think rather than relying on absolute file paths, it would be better to just import from `@fluentui/pkg-name/lib/Whatever`. Then add a `resolve.alias` configuration to the webpack config with the actual paths to the packages.

Reply:
Let me know if that's not clear and I'll go into more detail

Reply:
Fixed now. I tried to use the alias but didn't work on my local however importing it from the package and adding folder name works fine 👍